src/components/Account/Account.component.jsx

```diff
@@ -0,0 +1,30 @@
+// I believe this code ill be useful in the future

```

Comment:
this first line was very useful for me, great job leaving this!

I think it is better to leave a TODO list on this document with the steps that you can identify you will need and if you're not using it anywhere you could event uncomment the code.

Reply:
Thank you!

And you are right, I didn't think about that to be honest.